packages/react-router/modules/RouterContext.js

```diff
@@ -0,0 +1,8 @@
+import React from "react";
+import createReactContext from "create-react-context";
+
+const RouterContext = React.createContext
```

Comment:
That's already done in create-react-context https://github.com/jamiebuilds/create-react-context/blob/master/src/index.js

Reply:
Yep, this changed since the last time I touched this PR. Since I have to resolve a bunch of package-lock.json file conflicts anyways, I'll get this swapped out at that point.